whitepaper/Cosmology/wl.tex

```diff
@@ -10,152 +10,425 @@
-minimized.  A recommended metric is the count of fields with median
-RMS less then 0.8 or greater than 1.5 radians (these values to be
-reviewed again as additional experience is gained with additional
-OpSim schedule simulations and weak lensing analysis.)
-
-It is certain that actively controlling the statistics of RotSkyPos
-will require additional slewing of the camera rotator.  At present,
-the operations plan is to only slew when necessary to prepare for a
-filter change - that could be estimated at the equivalent of $\simeq
-3$ complete rotations per day.  \autoref{RotSkyPos} shows that to
-render the distribution completely uniform would require moving all
-observing angles an average of $\simeq 30$ degrees, or 300 complete
-rotations per night.  The timing of this has not been considered.
-Whether or not this uniformity could be achieved with less slew time
-if implemented in scheduling remains to be demonstrated.
-
-A similar metric for RotTelPos should be developed.
+The RotSkyPos metrics show that the majority of fields have good randomization
+of detector angles projected on the sky.  The randomization of parallactic
+angles is less successful, though this is to be expected due to fewer knobs
+available to adjust the parallactic angle of observations of a given field.  In
+both cases, however, a significant fraction of fields show metric values lower
+than expected for a uniform distribution.  Regardless of the \emph{per field}
+criterion adopted, it is desirable to avoid the incidence of individual
+discrepant fields.  The recommended criterion for randomization of RotSkyPos and
+parallactic angle is not the behavior of the majority of the fields, but of the
+minority with the least random behavior -- the number of non-random fields
+should be minimized.
+
+It is certain that actively controlling the statistics of RotSkyPos will require
+additional slewing of the camera rotator.  At present, the operations plan is to
+only slew (beyond that required to track the sky during exposures) when
+necessary to prepare for a filter change - that could be estimated at the
+equivalent of $\simeq 3$ complete rotations per night.  To engage the rotator by
+up to $\simeq 30$ degrees per visit would require $\simeq 300$ complete
+rotations per night, though it may be possible to increase isotropy with far
+less additional slewing.  The impact on survey efficiency and hardware wear and
+tear has not been considered.  Whether or not this uniformity could be achieved
+with less slew time if implemented in scheduling remains to be demonstrated.
+
+Increasing the isotropy of the parallactic angle is trickier, since the
+parallactic angle is only affected by the hour angle of observations (for a
+given field).  It may be possible, however, to adjust the scheduler cost
+function to better favor parallactic angle isotropy.
+
+In summary, image quality weighted randomization of RotSkyPos is required at
```

Comment:
Yes.   The assumptions were that the pixel level remap in DM pipline 
cannot correct to better than 3x the rms errors in the lab tests for 
dynamic and static CCD systematics.  There is a project document on this 
which I should reference.

On 6/29/16 7:48 PM, Rachel Mandelbaum wrote:
> In whitepaper/Cosmology/wl.tex
> <https://github.com/LSSTScienceCollaborations/ObservingStrategy/pull/478#discussion_r69061910>:
>
>> +additional slewing of the camera rotator.  At present, the operations plan is to
>> +only slew (beyond that required to track the sky during exposures) when
>> +necessary to prepare for a filter change - that could be estimated at the
>> +equivalent of $\simeq 3$ complete rotations per night.  To engage the rotator by
>> +up to $\simeq 30$ degrees per visit would require $\simeq 300$ complete
>> +rotations per night, though it may be possible to increase isotropy with far
>> +less additional slewing.  The impact on survey efficiency and hardware wear and
>> +tear has not been considered.  Whether or not this uniformity could be achieved
>> +with less slew time if implemented in scheduling remains to be demonstrated.
>> +
>> +Increasing the isotropy of the parallactic angle is trickier, since the
>> +parallactic angle is only affected by the hour angle of observations (for a
>> +given field).  It may be possible, however, to adjust the scheduler cost
>> +function to better favor parallactic angle isotropy.
>> +
>> +In summary, image quality weighted randomization of RotSkyPos is required at
>
> Might be good to summarize in a few words what assumptions are being
> made in this requirement (I mean, mitigation of systematics by changing
> observing strategy must assume that those systematics cannot be
> mitigated below a certain level in the software).
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/LSSTScienceCollaborations/ObservingStrategy/pull/478/files/68564bd57311f22658256026e5e6e6a39c2554f4#r69061910>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACA6Yr-h7hX-KIVSHvC0Ps2kWsXx_foeks5qQy50gaJpZM4I9hd->.
>